system-variables.md

```diff
@@ -269,11 +270,11 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Default value: `OFF`
 - This variable is used to set whether the `AUTO_INCREMENT` property of a column is allowed to be removed by executing `ALTER TABLE MODIFY` or `ALTER TABLE CHANGE` statements. It is not allowed by default.
 
-### tidb_analyze_version <span class="version-mark">New in v5.1.0</span>
+### tidb_analyze_version <span class="version-mark">New in v5.1</span>
```

Comment:
In fact, this variable was new in v5.1.0 rather than in a later v5.1.x version. Maybe we make it more specific, just as the other cases in this document do (specific to a patch version).
```suggestion
### tidb_analyze_version <span class="version-mark">New in v5.1.0</span>
```